jmh/build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java-library'
+    id 'nullaway.jacoco-conventions'
     id "me.champeau.jmh" version "0.6.6"
```

Comment:
Guess we have random versions across build files as it is, so maybe not an urgent issue...

Reply:
Yeah, this one I messed up before.  I should have listed it in the top-level `build.gradle` with `apply  false` and then just mentioned the id here.  I can do that in a separate PR

Reply:
#526

Reply:
Now fixed